scheduler/rank.go

```diff
@@ -252,6 +279,14 @@ OUTER:
 		normalizedFit := fitness / binPackingMaxFitScore
 		option.Scores = append(option.Scores, normalizedFit)
 		iter.ctx.Metrics().ScoreNode(option.Node, "binpack", normalizedFit)
+
+		// Score the device affinity
+		if devicesWithAffinities != 0 {
+			deviceAffinityScore /= float64(devicesWithAffinities)
```

Comment:
same comment as above, not sure why this is normalized by the number of affinities rather than sum of affinity weights